sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -393,6 +393,8 @@ def _group_to_dag_template(self, group, inputs, outputs, dependencies, pipeline_
               self.loops_pipeline[group_name]['spec']['params'].append({
                 'name': param[0], 'value': '$(params.%s)' % param[0]
               })
+      if sub_group.parallelism is not None:
```

Comment:
Do you need to check wether is it int type for the value parallelism?

Reply:
Thanks Feng，this parameter is a shared parameter with Argo, it's already validated in https://github.com/kubeflow/pipelines/blob/cc83e1089b573256e781ed2e4ac90f604129e769/sdk/python/kfp/dsl/_ops_group.py#L122-L126